app/index.js

```diff
@@ -10,7 +10,7 @@ import routes from './routes';
 import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import '../node_modules/toastr/build/toastr.min.css';
 import '../node_modules/griddle-react-bootstrap/dist/griddle-react-bootstrap.css';
-import '../ux/dev/css/styles.css';
+// import '../ux/dev/css/styles.css'; //ToDo: commenting MSUIX styles until fully integrated.
```

Comment:
It's the same. I may prefer leaving this way with the ToDo so the UI guys (and we) remember that thing that's needs to be done. 
Thanks anyway!